tt_metal/hw/firmware/src/brisc.cc

```diff
@@ -168,6 +193,11 @@ void device_setup() {
     // FIXME MT: enable later
     // enable_power_management();
 
+#ifdef ARCH_BLACKHOLE
+    // Disable DEST CG
+    *((uint32_t volatile*)RISCV_DEBUG_REG_DEST_CG_CTRL) = 0;
```

Comment:
not familiar with Dest CG, why do we need to do this?

Reply:
ttmtrajkovic has mentioned that this clock gating register is on by default for blackhole, but it should be off for all devices, until software enables it.

Reply:
should this be ifndef?

Reply:
No it needs to be disabled for blackhole?

Reply:
oh sorrry misread your original comment, thought you said it should be off for all `other` devices. ignore me :)